### DISCUSSION TITLE
Next Zoomed Split

### DISCUSSION THREAD

**Discussion by danieluhl** [ghostty-org/ghostty#4177] [Feature Requests, Ideas] [answered]
Something I miss from kitty is stacked layout where I can go to the next window (zoomed split in ghostty terms) without losing the fullscreen "zoomed" experience. Currently ghostty bounces out of zoom when you jump to or rotate through splits.

This is important when working on a laptop or small screen where I don't want to constantly be zooming back in, but I do want to bounce between terminal and vim for each of a few open projects (which I keep one per tab).

**Comment by michix** [comment, marked as answer]
Answered in #3484
